app/src/main/java/ch/epfl/sweng/swenggolf/profile/PointType.java

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.sweng.swenggolf.profile;
+
+/**
+ * Enumeration for the points type.
+ */
+public enum PointType {
+
+    POST_OFFER(10),
+    RESPOND_OFFER(20),
+    CLOSE_OFFER(50);
```

Comment:
It's very nice that you've implemented it.

I think it would be a good idea to discuss it on your issue. It's so we can all agree on what values we should give each bonus, and maybe add new bonuses.

Reply:
Yes of course it was just an example of what we can do!